proto/plugin.proto

```diff
@@ -406,6 +410,7 @@ message StatusReport {
     ALIVE = 1;
     READY = 2;
     DOWN = 3;
+    MISSING = 5; // We're expecting it to exist, but it does not.
```

Comment:
Should this go down one to keep the var order?

Reply:
This order made sense to me, because PARTIAL is always treated as our last, catch-all status when you have some non-uniform combination of our other statuses. From what I can see, our convention in these proto files is to make the order read logically at the expense of the numbers not being sequential, because the order in the file doesn't matter to protoc.